### ISSUE TITLE
Workflow Editor Stops working when post_job_actions is set

### ISSUE THREAD

**Issue by hepcat72** [galaxyproject/galaxy#6918]
I have isolated this issue to workflow step 21, specifically, the "post_job_actions" in that step.  I was editing the cut tool outfile attributes, specifying which column was the chromosome, start, stop, strand, and name and the editor hung when I tried to save it.  After that, the workflow would not load.  I downloaded the workflow and on a hunch, removed the "post_job_actions" stuff from step 21 (cut tool) and the workflow loaded fine.  I noted that there was only 1 setting in the post_job_actions section to set the chromosome column.  So that must be where the first error occurred.

My intent at the time was to be able to connect the output of cut to the input of closestBed.  I *should* have changed the output type, but being somewhat inexperienced, I first came across the setting of these columns, thinking that would allow me to connect the 2 tools...

Here's a gist of the workflow that does not load:

[galaxy-workflow-chip-seq_single_end_-_step_2__peaks_and_motifs-ga](https://gist.github.com/hepcat72/f1d9765d6a543666f211ede27592f977#file-galaxy-workflow-chip-seq_single_end_-_step_2__peaks_and_motifs-ga)

I have recovered my workflow and fixed the issue by manually editing the json of the workflow, so I'm not waiting on a fix.  Thus, this is a low priority issue.  I just wanted to report it so that you could correct the issue for anyone else that might run into it.

**Comment by dannon** [comment]
@hepcat72 Thanks for reporting this; I have a hunch as to what the problem is and will update here once tested/fixed.

**Comment by dannon** [comment]
@hepcat72 Sorry for the delay here, got stuck working on other stuff.  #6955 should fix this for you.

**Comment by mvdbeek** [comment]
https://github.com/galaxyproject/galaxy/pull/6955 is merged and should fix this, if not feel free to re-open.
